bin/build.sh

```diff
@@ -37,7 +37,7 @@ EOF
 }
 
 fromPom() {
-  mvn -f $1/pom.xml -P$2 help:evaluate -Dexpression=$3 -B \
+  mvn ${WASABI_MAVEN} -f $1/pom.xml -P$2 ${WASABI_MAVEN} help:evaluate -Dexpression=$3 -B \
```

Comment:
great observation ... i opted to front the mvn invocation with the optional env-var overrides, yet neglected to remove the original ref.